src/lib/components/Carousel/Carousel.test.tsx

```
@@ -156,9 +156,24 @@ const nonActiveIndicatorClasses =
   'bg-white/50 hover:bg-white dark:bg-gray-800/50 dark:hover:bg-gray-800';
 
 const items = [
-  <img src='https://flowbite.com/docs/images/carousel/carousel-1.svg' />,
-  <img src='https://flowbite.com/docs/images/carousel/carousel-2.svg' />,
-  <img src='https://flowbite.com/docs/images/carousel/carousel-3.svg' />,
-  <img src='https://flowbite.com/docs/images/carousel/carousel-4.svg' />,
-  <img src='https://flowbite.com/docs/images/carousel/carousel-5.svg' />,
+  <img
+    src='https://res.cloudinary.com/supertramp69420/image/upload/v1738449525/carousel-1_n3vlkd.svg'
+    alt='Slide 1'
+  />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)